webapp/components/admin_console/storage_settings.jsx

```diff
@@ -258,6 +262,43 @@ export default class StorageSettings extends AdminSettings {
                     onChange={this.handleChange}
                     disabled={!this.state.enableFileAttachments}
                 />
+                <BooleanSetting
+                    id='EnableClientSideEncryption'
+                    label={
+                        <FormattedMessage
+                            id='admin.file.enableClientSideEncryptionTitle'
+                            defaultMessage='Encrypt Files at the Mattermost server'
```

Comment:
@jasonblais We need some review on this text. We should at least make it clear that this only affects Amazon S3 file storage, and probably group it with the other S3 settings. Should that be done after this PR is merged, or do we want to do it now?